game.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'environment'
+
+class Game 
+  attr_accessor :depot, :control
+  include Environment
+
+  def initialize
+    @board = { payers: {}, control: {}, status: nil}
+    @depot = Environment::Depot.new
+    @control = Environment::Control.new
+  end
+
+  def new_worker(name:)
+    worker = Environment::Worker.new(name:name, personal_data: nil,  professional_data: nil)
+    @board[:payers[name]] = worker
+    
+  end
+
+  def new_mission(name:, objective: , pack: )
```

Comment:
I am curious, did this line worked? I would add at least nil when intializing arguments, something like.
`def new_mission(name:, objective: nil, pack: nil)`

Reply:
Yes, It didn't show me any warning or error